jobs/async-upload/tests/test_config.py

```diff
@@ -0,0 +1,211 @@
+import os
+import random
+import tempfile
+import shutil
+from pathlib import Path
+import pytest
+from job.config import get_config
+
+MR_PREFIX = "MODEL_SYNC"
+MR_SOURCE_PREFIX = "MODEL_SYNC_SOURCE"
+MR_DEST_PREFIX = "MODEL_SYNC_DESTINATION"
+
+
+@pytest.fixture
+def env_vars():
+    """Fixture to set up environment variables for testing"""
+    # Save original environment
+    original_env = dict(os.environ)
+
+    os.environ["MODEL_SYNC_SOURCE_TYPE"] = "oci"
+    os.environ["MODEL_SYNC_SOURCE_OCI_URI"] = "quay.io/example/models"
+
+    # Set up test environment variables
+    os.environ["MODEL_SYNC_DESTINATION_TYPE"] = "s3"
+    os.environ["MODEL_SYNC_DESTINATION_AWS_ACCESS_KEY_ID"] = "destination_key_env"
+    os.environ["MODEL_SYNC_DESTINATION_AWS_SECRET_ACCESS_KEY"] = (
+        "destination_secret_env"
+    )
+    os.environ["MODEL_SYNC_DESTINATION_AWS_REGION"] = "us-east-1"
+    os.environ["MODEL_SYNC_DESTINATION_AWS_BUCKET"] = "destination-bucket-env"
```

Comment:
good to know it's possible in the future and as a config test case, I just want to reiterate this should not be the first focus of the implementation (and the other way around S3 -> OCI)